subsys/bluetooth/mesh/light_ctrl_srv.c

```diff
@@ -524,10 +524,11 @@ static void prolong(struct bt_mesh_light_ctrl_srv *srv)
 static void ctrl_enable(struct bt_mesh_light_ctrl_srv *srv)
 {
 	atomic_clear_bit(&srv->flags, FLAG_RESUME_TIMER);
+	atomic_clear_bit(&srv->flags, FLAG_AMBIENT_LUXLEVEL_SET);
 	srv->lightness->ctrl = srv;
 	BT_DBG("Enable Light Control");
 	transition_start(srv, LIGHT_CTRL_STATE_STANDBY, 0);
-	reg_start(srv);
+	/* Regulator remains stopped until fresh LuxLevel is received. */
```

Comment:
lux level, luxlevel or LuxLevel? I've seen all three in use.
Which one should we stick to?

Reply:
Other comment in this file is using LuxLevel, so I will stick to that here?